practices/simpleDesign/readme.md

```diff
@@ -0,0 +1,35 @@
+# Simple Design
+
+The simplest design is often (almost always) the best solution to a given problem. The simplest solution is the easiest to maintain and the easiest to change (assuming you have appropriate testing of course). The assertion here is that trying to "build for the future" or "build in extensibility" ends up being more costly as we cannot predict what future requirements will be. Solve for the now, iterate on future later.
+
+## The What and the Why
+
+### The What
+
+Simple design refers to designing your code in a simple, clean and elegant way, but not necessarily simplistic. The following are guidelines to simple design:
```

Comment:
'Simple design refers to designing your code in a simple, clean and elegant way, but not necessarily simplistic.' sounds wrong... not sure if it actually violates a real agreement rule, or if it's just one of the rule like 'the three cute puppies' is right and 'the cute three puppies' is wrong because of more colloquial rules. Either way 'Simple design refers to designing your code in a simple, clean and elegant way, that is not necessarily simplistic.' seems better. Absent context the distinction that we are not advocating simplistic design can probably be removed.

Reply:
Okay! Removed distinction between simple and simplistic.